infra/gcp/bash/ensure-staging-storage.sh

```diff
@@ -378,7 +394,13 @@ function ensure_release_manager_special_cases() {
         # ref: https://github.com/kubernetes/release/pull/1230
         if [[ "${project}" == "k8s-staging-kubernetes" ]]; then
             color 6 "Empowering kubernetes-release-test GCB service account to admin GCR"
-            empower_svcacct_to_admin_gcr "648026197307@cloudbuild.gserviceaccount.com" "${project}"
+            local serviceaccount="648026197307@cloudbuild.gserviceaccount.com"
+            empower_svcacct_to_admin_gcr "${serviceaccount}" "${project}"
+
+            ensure_signing_service_account \
```

Comment:
we only use one GCB SA for all the staging project.

Reply:
As we discussed during RelEng meeting, I suggest we stop use default service accounts for the release process, artifacts signatures and promotion. And your observation is correct; since use separate projects, we rely on those default service accounts.

Reply:
@ameukam is it even possible to create a new `@cloudbuild.gserviceaccount.com` service account without copying all the permissions: https://cloud.google.com/build/docs/securing-builds/configure-user-specified-service-accounts

Reply:
@saschagrunert  we can't create `cloudbuild.gserviceaccount.com` service accounts but my proposal implies we get GCP service accounts and ensure we have the right permissions (or more) for those SA and it can be done if we do the correct bindings.

Reply:
@ameukam do you think you can support us with the scripting changes here? (cc @cpanato) :blush:

Reply:
@saschagrunert Yes of course. I'm out of office but I'll update the issue related to break down what I'm saying.